pkg/logging/options.go

```diff
@@ -1,25 +1,26 @@
 package logging
 
-import "github.com/stackrox/rox/pkg/notifications"
+import "github.com/stackrox/rox/pkg/administration/events"
 
 // options for the logger.
 type options struct {
-	notificationStream    notifications.Stream
-	notificationConverter notifications.LogConverter
+	AdministrationEventsConverter events.LogConverter
+	AdministrationEventsStream    events.Stream
 }
 
 // OptionsFunc allows setting log options for a logger.
 type OptionsFunc = func(option *options)
 
-// EnableNotifications enables the logger to send log statements of
-// Errorw and Warnw as notifications to the end-user.
+// EnableAdministrationEvents enables the logger to send log statements of
+// Errorw and Warnw as administration events to the end-user.
 //
 // Before enabling logging for your package, ensure that:
-// * your module resolves to a specific domain (see pkg/notifications/domain.go).
-// * notifications emitted from your specific package have hints defined to help users (see pkg/notifications/hints.go).
-func EnableNotifications() OptionsFunc {
+//   - your module resolves to a specific domain (see pkg/administration/events/domain.go).
+//   - Administration events emitted from your specific package have hints defined to help
+//     users (see pkg/administration/events/hints.go).
+func EnableAdministrationEvents() OptionsFunc {
```

Comment:
Heavy preference for EnableAdministrationEvents to pass the events.Datastore interface here instead of calling the singleton from within. This allows the singleton to be moved out of the package

Reply:
Will address this in a follow-up to ensure that the `Singleton()` call will be done _once_ instead of on each `EnableAdministrationEvents` call.

Reply:
Did this within #7597 .